.gitignore

```diff
@@ -12,3 +12,4 @@ node_modules/
 /.repl-history
 .vscode
 /junit-reports/
+config/development.json
```

Comment:
I think correct path here would be `/config/development.json`

Reply:
If the intention is to use `config/test.json`, then could `process.env.NODE_CONFIG_ENV = 'test'` be used instead?

Could this setting be moved to `Makefile` to avoid affecting other mocha tests (e.g. e2e tests) which already use their own `NODE_CONFIG_ENV`?

Reply:
done